store/storetest/store.go

```diff
@@ -45,6 +47,7 @@ type Store struct {
 	LinkMetadataStore         mocks.LinkMetadataStore
 }
 
+func (s *Store) CacheProvider() cache.CacheProvider                { return &s.Cache }
```

Comment:
Not needed, the store itself doesn't need to know anything about the Cache Provider

Reply:
Fixed